docs/userguide/globalview/proxy-global-resource.md

```diff
@@ -51,7 +41,37 @@ spec:
       kind: Node
 ```
 
-After applying it, you can access pods and nodes with kubectl. Enjoy it!
+```shell
+$ kubectl --context karmada-apiserver apply -f resourceregistry.yaml
+```
+
+Based on the ResourceRegistry above, you can access pods and nodes with proxy function.
+
+### Step2: Access by proxy
+
+Append `/apis/search.karmada.io/v1alpha1/proxying/karmada/proxy` to the server address of `karmada-proxy.config`. Then set this file as default config:
+
+```shell
+export KUBECONFIG=$HOME/karmada-proxy.config
+```
+
+After processing the above steps, you can access pods and nodes with kubectl.
+
+Taking getting the pod log as an example, you can use the following command:
+
+```shell
+$ kubectl get --raw /apis/search.karmada.io/v1alpha1/proxying/karmada/proxy/api/v1/namespaces/<ns>/pods/<pod-name>/log
```

Comment:
Is this still working with the KUBECONFIG that already append ``/apis/search.karmada.io/v1alpha1/proxying/karmada/proxy``?

Reply:
Yes, it is.
![image](https://github.com/karmada-io/website/assets/78244870/a96af31b-cf67-40e6-b4ee-d7cf4e47ded7)